sugargame/__init__.py

```diff
@@ -1 +1 @@
-__version__ = '1.1'
+__version__ = '1.2'
```

Comment:
hmm. why not `1.3`?

Reply:
The activity is still in python2 and version 1.3 of sugargame uses python3

Reply:
Agreed.  v1.3 includes Port to Python 3.